test/e2e/network/networking.go

```diff
@@ -179,6 +179,17 @@ var _ = SIGDescribe("Networking", func() {
 			config.DialFromTestContainer("udp", config.NodeIP, config.NodeUDPPort, config.MaxTries, 0, config.EndpointHostnames())
 		})
 
+		ginkgo.It("should function for pod-Service: sctp [Feature:SCTPConnectivity][Disruptive]", func() {
```

Comment:
I think this needs a comment explaining why it's Disruptive, except that in fact, it *isn't* Disruptive at this point since the test that it disrupts hasn't been written yet... So maybe just don't mark them `[Disruptive]` for now, and that can be done as part of the PR that adds the other tests.

Reply:
My understanding from here https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/0015-20180614-SCTP-support.md#sctp-connectivity-tests is that the reason for these tests to be `[Disruptive]` is because they will load the SCTP module kernel. And this is happening as soon as we run `NewNetworkingTestConfig` with `config.SctpEnabled = true`, as it will pass the `--sctp-port` parameter to the agnhost image. That's also the reason why I cared about enabling it only in these tests instead of having it enabled by default.

Reply:
"Disruptive" means that it needs to be run serially because it might cause other tests to fail if it was run at the same time as them. The KEP talks about marking this test as disruptive because it was assuming that we would have added the other tests first. But since we haven't added those tests yet, then there are no tests that would fail if the sctp module was unexpectedly loaded while they were running, so this test isn't *currently* disruptive.

Reply:
Ah, thanks for taking the time to clarify this, now I understand your previous comment. 
Will remove the [Disruptive] label and add a comment specifiying the need for it once the other tests are being added.